### PR TITLE
UpdateValueCore for boolean node

### PR DESCRIPTION
**Pull request by marimano**
It was lost during some merging

### CODE REVIEW

**Comment by marimano** [comment]
Hi @Benglin, please take a look. This PR is small but important :)

File: src/Libraries/CoreNodesUI/Input/BoolSelector.cs [hunk 1]
```diff
@@ -23,6 +23,17 @@ protected override bool DeserializeValue(string val)
...
+        {
+            if (name == "Value")
+            {
+                Value = DeserializeValue(value);
+                return true;
+            }
+
+            return base.UpdateValueCore(name, value);
+        }
+
```
Comment (Benglin) [inline src/Libraries/CoreNodesUI/Input/BoolSelector.cs]: Hi @marimano, this looks reasonable but I don't see it in the latest master, is this being called only from the web server when user toggles `True` or `False` on the client?

Comment (marimano) [inline src/Libraries/CoreNodesUI/Input/BoolSelector.cs]: @Benglin, yes, it's being called only from DynamoWebServer when Flood needs to change boolean node's value. These changes were added by me [here](https://github.com/Benglin/Dynamo/pull/231) but after big merge I don't see them anymore

**Comment by pboyer** [comment]
LGTM, this is basically the same code as before.

**Comment by Benglin** [comment]
Thanks @marimano and @pboyer!

**Pull request merged**